source/learners/sfd_discussions/discussions_notifications.rst

```diff
@@ -0,0 +1,104 @@
+.. _staying-updated-with-notifications:
+
+##################################
+Staying Updated with Notifications
+##################################
+
+Discussion notifications keep you informed when others interact with your posts or when instructors share updates.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+There are four notification preferences for Discussions:
+
+1. **Activity notifications**: Cover all activity related to your own posts, responses, and followed threads.  
+2. **New question posts**: Alerts you when someone creates a new question type post in your course.
+3. **New discussion posts**: Alerts you when someone creates a new discussion type post.
+4. **New posts from instructors**: Notifies you when an instructor or staff member creates a post.
+
+The tables below shows who receives each notification by default.
```

Comment:
The tables are a little confusing to me. How do the two tables relate to the four preferences above?

Reply:
I've merged both tables into one. I had split tables to avoid repetition of the preference named "Activity notifications". But unfortunately i can't get to merge cells in an rst table.